README.md

```diff
@@ -0,0 +1,20 @@
+# Custom floating label style input field for iOS with swift
+floating label is a text label which appears inside the input field at full font-size. When interacted with, the label “floats” above, making room for the user to input a value
+
+## Features
+* Float label pattern
+* Custom text input
+* Text input validation
+
+## Requirement
+* iOS
+* compatible using swift
+
+## Get started with custom floating label style input
+* Create a swift file to hold our customization code.
+* We create our text field in our storyboard and assign it’s class to our custom 'UITextField'
```

Comment:
* Copy `FloatingLabelinput.swift` to your project.
* Assign the `FloatingLabelinput` as a custom class to your `UITextField` in storyboard.